### ISSUE TITLE
Icon don't follow dark and light theme

### ISSUE THREAD

**Issue by aksharlabs** [heera-os/heera-icon-theme#1]
In dark mode icons are fine everywhere but in light mode some ui icons are not visible.and icon theme doesn't work with gtk apps.

**Comment by khumnath** [comment]
Yes there is only one varient of icon theme now.we are working on it.
